eightshift-forms.php

```diff
@@ -28,30 +28,52 @@
 }
 
 /**
- * Include the autoloader so we can dynamically include the rest of the classes.
+ * Bailout, if the plugin is not loaded via Composer.
+ */
+if (!\file_exists(__DIR__ . '/vendor/autoload.php')) {
+	return;
+}
+
+/**
+ * Require the Composer autoloader.
  */
 $loader = require __DIR__ . '/vendor/autoload.php';
-require __DIR__ . '/vendor-prefixed/autoload.php';
 
 /**
- * The code that runs during plugin activation.
+ * Require the Composer autoloader for the prefixed libraries.
  */
-\register_activation_hook(
-	__FILE__,
-	function () {
-		PluginFactory::activate();
-	}
-);
+if (\file_exists(__DIR__ . '/vendor-prefixed/autoload.php')) {
+	require __DIR__ . '/vendor-prefixed/autoload.php';
+}
```

Comment:
Should a check for the existence of this file also be checked against and returned early if it's missing like for the regular autoloader?